sharktank/sharktank/layers/paged_attention.py

```diff
@@ -126,6 +128,23 @@ def __init__(
         self.cache_dtype = cache_dtype
         self.attn_dtype = attn_dtype
 
+    def unflatten_page_table(
+        self,
+        state: torch.Tensor | SplitPrimitiveTensor | ReplicatedTensor,
+        pipeline: int,
+    ) -> torch.Tensor | SplitPrimitiveTensor | ReplicatedTensor:
```

Comment:
Can you document what the difference between `unflatten_page_table` and `unflatten_page_tables` is? Why do we have multiple page tables? Is one for the unsharded version? This needs more documentation.